typescript/packages/lookslike-high-level/src/recipes/iframeExample.ts

```diff
@@ -0,0 +1,89 @@
+import { html } from "@commontools/common-html";
+import {
+  recipe,
+  fetchData,
+  UI,
+  NAME,
+  ifElse,
+  lift,
+  generateData,
+  handler,
+} from "@commontools/common-builder";
+import { cell} from "@commontools/common-runner";
+
+const formatData = lift(({ obj }) => {
+  console.log('stringify', obj)
+  return JSON.stringify(obj, null, 2)
+})
+
+const tap = lift(x => {
+  console.log(x);
+  return x;
+})
+
+// TODO(ben): this is the shared state with the iframe, unsure if this is the right way to orchestrate
+const context = cell({ exampleKey: 456 })
+const data = context.getAsProxy()
+const getHtml = lift((obj) => obj?.html)
```

Comment:
this shouldn't be necessary, you can just say `obj.html` below, i.e. just `response.result.html`.